### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.4.7][Eclipse][Wipro]:Keyboard focus is not visible for App Service Details Pane in deploy as web app

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1370]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Publish as Azure web app.
4. In "Deploy web app" dialog will open. 
5. In "Deploy web app" dialog , navigate to App Service Details pane then observe the keyboard focus

**Expected Result:**
Keyboard focus should be visible for App Service Details pane

**Actual Result:**
Keyboard focus is not visible for App Service Details pane

**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
**OSBuild Version**: Version 1709(Os Build 16299.192)

![keyboard focus is not visible for app service details pane](https://user-images.githubusercontent.com/22956130/35680564-bb7ee726-0780-11e8-8ea0-489eb89b9174.jpg)

**Comment by jdneo** [comment]
Caused by the `Browser` widget behavior.

Same behavior in Eclipse's official Brwoser page:
![image](https://user-images.githubusercontent.com/6193897/38720067-68913a76-3f27-11e8-9410-ebdcc6be5288.png)

**Comment by Eskibear** [comment]
tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=533978

**Comment by Eskibear** [comment]
linked to #1619

**Comment by Eskibear** [comment]
* **Importance of experience**: minimal. Very low usage based on telemetry.
* **Quality of experience**: not impacted. Users can view the app service details in Azure Portal. 
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

See:
![image](https://user-images.githubusercontent.com/2351748/46596756-2c6c2400-cb10-11e8-84cd-6646a0264979.png)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749820
